### PR TITLE
feat: newline after block required

### PR DESCRIPTION
**Pull request by kirjavascript**
lint

### CODE REVIEW

**Review by d3lm — COMMENTED** [review]
Awesome! It'd be nice if we also added tests for this new rule.

**Comment by kirjavascript** [comment]
hmm.. cant quite figure out how to get the type definitions to work

types

<s>only the `dist` directory and not the `src` seem to appear in node_modules so no type definitions are available for this</s>

ok I think I get it I have to define them myself

**Comment by d3lm** [comment]
@kirjavascript If you need help, I can have a look with you tomorrow.

File: src/configs/recommended.ts [hunk 1]
```diff
@@ -81,7 +86,11 @@ export = {
         prev: 'case',
         next: 'default',
       },
-      { blankLine: 'any', prev: '*', next: 'break' },
+      {
```
Comment (d3lm) [inline src/configs/recommended.ts:89]: 🪴 Unrelated formatting fix.

**Review by d3lm — APPROVED** [review]
